examples/quickstart/jupyter-notebooks/notebooks/03-query/04-UnionOperations.ipynb

```diff
@@ -0,0 +1,644 @@
+    "  WHERE Origin = 'SFO'\n",
+    "  AND DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  ),\n",
+    "set2 AS (\n",
+    "SELECT\n",
+    "  __time,\n",
+    "  \"Origin\",\n",
+    "  \"Tail_Number\",\n",
+    "  \"Flight_Number_Reporting_Airline\"\n",
+    "  FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "  WHERE Origin = 'ATL'\n",
+    "  AND DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  )\n",
+    "\n",
+    "SELECT __time,\n",
+    "    \"Origin\",\n",
+    "    \"Tail_Number\",\n",
+    "    \"Flight_Number_Reporting_Airline\"\n",
+    "FROM (\n",
+    "    SELECT * from set1\n",
+    "    UNION ALL\n",
+    "    SELECT * from set2\n",
+    "    )\n",
+    "ORDER BY __time\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25001794-e1a7-4325-adb3-2b8f26036261",
+   "metadata": {},
+   "source": [
+    "Notice the `ClientError` that Druid returns. Rather than simply concatenating two `TABLE`s of results together, Druid switched to table level `UNION ALL`, and with it a switch to [`UNION`](https://druid.apache.org/docs/26.0.0/querying/datasource.html#union) datasources. These cannot be used in the same way as `TABLE`s; there are constraints on what can be done with them. An `EXPLAIN PLAN` for these types of queries shows this in detail:\n",
+    "\n",
+    "```json\n",
+    "\"type\": \"union\",\n",
+    "\"dataSources\": [\n",
+    "    {\n",
+    "    \"type\": \"table\",\n",
+    "    \"name\": \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    },\n",
+    "    {\n",
+    "    \"type\": \"table\",\n",
+    "    \"name\": \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    }\n",
+    "```\n",
+    "\n",
+    "In large deployments of Druid, data engineers may decide to break up a single data set into multiple `TABLE`s. Some reasons for this might be:\n",
+    "\n",
+    "* `TABLE`s separated by their `__time` granularity (ie. the level of roll-up)\n",
+    "* `TABLE`s logically separated for security reasons by tenant\n",
+    "* `TABLE`s broken up by different filter conditions\n",
+    "* `TABLE`s separated by upstream data source (e.g. different Kafka topics for different cloud regions, different groups of source device, and so on)\n",
+    "* `TABLE`s that break the entire data set up into different periods of time, perhaps with different retention periods\n",
+    "\n",
+    "The next two cells to create two new tables, `wikipedia-en` and `wikipedia-fr`. One table contains only English language channels, while the second only contains French language channels. Imagine that this is a design decision taken by a data engineer so that they can be governed separately. But now we want to have a single view of both that people can query.\n",
```

Comment:
```suggestion
    "The next two cells create two new tables, `wikipedia-en` and `wikipedia-fr`. One table contains only English language channels, while the second contains only French language channels. Imagine that this is a design decision taken by a data engineer so that they can be governed separately. But now we want to have a single view of both that people can query.\n",
```